src/lib/dataAccess.js

```diff
@@ -15,16 +15,13 @@ export const httpGet = (endPoint) => {
         });
 };
 
-export const httpPost = (endPoint, accessToken, body) => {
+export const httpPost = (endPoint, body) => {
     const url = baseURL + endPoint;
 
     return new Promise((resolve, reject) => {
         axios
             .post(url, body, {
-                headers: {
-                    Authorization: "Bearer " + accessToken,
-                    "Content-Type": "application/json",
-                },
+                headers: {},
```

Comment:
Interesting issue that I found. Axios seems to set a default header for `content-type`, which the backend does not allow (only allows `x-requested-with` from what I understand). I had to manually set empty headers but perhaps this may require some backend/frontend changes. Thoughts @jaydhulia @ahmedhamodi

Reply:
I'm okay to set empty headers, but this is def more of a Jay question so will defer to him.

Reply:
We can update the backend’s allowed headers to include this header (content-type), it’s a pretty standard header. We will also need to update this PR (and the backend allowed headers) to allow CSRF token once that’s merged in

Reply:
Addressed content-type with a proposed change here: https://github.com/uwblueprint/cdc-backend/pull/84

@ahmedhamodi should I address CSRF now or only when Jay merges?

Reply:
I'm actually not sure what work is needed to address CSRF - if it's minimal I say go for it now @aaronabraham311

Reply:
Let's do it once Jay merges his PR because I'm not quite sure what I would need to do as of now. Would like this merged as an example since @andstun will be working on this soon